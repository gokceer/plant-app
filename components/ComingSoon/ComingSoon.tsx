import React from "react";
import { Image, SafeAreaView, StyleSheet, Text } from "react-native";

const ComingSoon = () => {
  return (
    <SafeAreaView style={styles.container}>
      <Image
        source={require("../../assets/images/plantAppLogo.png")}
        style={styles.image}
      />
      <Text style={styles.text}>Coming Soon ðŸš€</Text>
    </SafeAreaView>
  );
};

export default ComingSoon;

const styles = StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: "center",
    alignItems: "center",
  },
  image: {
    width: 100,
    height: 100,
    borderRadius: 10,
    marginBottom: 12,
  },
  text: {
    fontFamily: "RubikMedium",
    fontSize: 16,
  },
});
