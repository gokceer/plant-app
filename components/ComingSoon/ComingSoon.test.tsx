import { render } from "@testing-library/react-native";
import React from "react";
import ComingSoon from "./ComingSoon";

describe("ComingSoon Component", () => {
  it("renders correctly and matches snapshot", () => {
    const { toJSON } = render(<ComingSoon />);
    expect(toJSON()).toMatchSnapshot();
  });
});
