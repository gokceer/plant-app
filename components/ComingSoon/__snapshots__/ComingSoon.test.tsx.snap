// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComingSoon Component renders correctly and matches snapshot 1`] = `
<RCTSafeAreaView
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Image
    source={
      {
        "testUri": "../../../assets/images/plantAppLogo.png",
      }
    }
    style={
      {
        "borderRadius": 10,
        "height": 100,
        "marginBottom": 12,
        "width": 100,
      }
    }
  />
  <Text
    style={
      {
        "fontFamily": "RubikMedium",
        "fontSize": 16,
      }
    }
  >
    Coming Soon ðŸš€
  </Text>
</RCTSafeAreaView>
`;
